src/Sparrow/LowMemory/MemoryInformation.cs

```diff
@@ -165,7 +164,7 @@ public static long GetRssMemoryUsage(int processId)
             }
         }
 
-        public static (long MemAvailableInKb, long SwapTotalInKb, long CommitedInKb, long TotalMemoryInKb) GetFromProcMemInfo()
+        public static (long MemAvailable, long SwapTotal, long Commited, long TotalMemory) GetFromProcMemInfo()
```

Comment:
Why dropping unit suffix? Can we use `Size` struct here instead?

Reply:
The caller works with bytes

Reply:
You know that but someone changing this in the future code will need to figure it out. Putting this as `MemAvailableInBytes` will make it clearer. Using `Size` here would make it more flexible regardless the unit that the caller works with

Reply:
Fixed

Reply:
Great. Thanks